client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -203,12 +227,29 @@ class Upload extends React.Component {
 	}
 }
 
+const ConnectedUpload = connectOptions(
+	( props ) => (
+		<Upload { ...props } />
+	)
+);
```

Comment:
I think `( props ) => <Component { ... props } />` is essentially the identity HoC, so you can probably shorten this to

```js
const ConnectedUpload = connectOptions( Upload )
```